org.clawsmail.claws-mail.json

```diff
@@ -0,0 +1,161 @@
+{
+  "app-id": "org.clawsmail.Claws-Mail",
+  "runtime": "org.gnome.Platform",
+  "runtime-version": "3.32",
+  "sdk": "org.gnome.Sdk",
+  "command": "claws-mail",
+  "finish-args": [
+    "--socket=x11",
+    "--socket=wayland",
+    "--share=ipc",
+    "--share=network",
+    "--filesystem=~/.claws-mail:create",
```

Comment:
I would suggest opening an issue upstream to use `$XDG_CONFIG_HOME` instead of hardcoding paths. This will allow the app to keep those app in safe place where other apps can't look at.

Reply:
I have a question outstanding regarding which other directories are still needed. I might need to generalize to `--filesystem=home`, as there is no way to predict where the user will store his mail-directory. Will follow up.